src/Test.sol

```diff
@@ -2,10 +2,25 @@
 pragma solidity >=0.6.2 <0.9.0;
 
 import {CommonBase} from "./Common.sol";
-import "ds-test/test.sol";
-// forgefmt: disable-next-line
-import {console, console2, StdAssertions, StdChains, StdCheats, stdError, stdJson, stdMath, StdStorage, stdStorage, StdUtils, Vm} from "./Components.sol";
+import {DSTest} from "ds-test/test.sol";
+import {console} from "./console.sol";
+import {console2} from "./console2.sol";
+import {StdAssertions} from "./StdAssertions.sol";
+import {StdChains} from "./StdChains.sol";
+import {StdCheats} from "./StdCheats.sol";
+import {stdError} from "./StdError.sol";
+import {stdJson} from "./StdJson.sol";
+import {stdMath} from "./StdMath.sol";
+import {StdStorage, stdStorage} from "./StdStorage.sol";
+import {StdUtils} from "./StdUtils.sol";
+import {Vm} from "./Vm.sol";
 
-abstract contract TestBase is CommonBase {}
+abstract contract TestBase is CommonBase {
+    // Default address for tx.origin and msg.sender, 0x1804c8AB1F12E6bbf3894d4083f33e07309d1f38.
+    address internal constant DEFAULT_SENDER = address(uint160(uint256(keccak256("foundry default caller"))));
+
+    // Address of the test contract, deployed by the DEFAULT_SENDER.
+    address internal constant DEFAULT_TEST_CONTRACT = 0x5615dEB798BB3E4dFa0139dFa1b3D433Cc23b72f;
```

Comment:
I'd say just keep these in `CommonBase` as they are constants of forge, and afaik apply to scripts too,. So it's easier to track and find them all when they're in a single spot, and you typically want more than one (e.g. when `vm.assume`ing to exclude all forge addresses)